src/json_file_module.F90

```diff
@@ -342,6 +350,22 @@ subroutine get_json_core_in_file(me,core)
     end subroutine get_json_core_in_file
 !*****************************************************************************************
 
+
+!*****************************************************************************************
+!>
+!  Get a pointer of the [[json_value]] in this [[json_file]].
+
+    subroutine get_json_value_in_file(me,value)
```

Comment:
I guess you're right, calling the generic ``get`` with a ``json_value`` argument will do the job